docker: Error response from daemon: Conflict. The container name "/pg-docker" is already in use by container "5797dde0cd549531edb3910d30dde28692ee8a67001848643364e40f51199073". You have to remove (or rename) that container to be able to reuse that name.
See 'docker run --help'.
docker: Error response from daemon: Conflict. The container name "/pg-docker" is already in use by container "23815e72583ef6745c72b8b963c29313b37c5ee2612ca7c21c0a6d424157b91c". You have to remove (or rename) that container to be able to reuse that name.
See 'docker run --help'.
The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data ... ok
creating subdirectories ... ok
selecting dynamic shared memory implementation ... posix
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting default time zone ... Etc/UTC
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok


Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data -l logfile start

initdb: warning: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
waiting for server to start....2021-12-28 15:24:00.060 UTC [48] LOG:  starting PostgreSQL 13.5 (Debian 13.5-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
2021-12-28 15:24:00.062 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2021-12-28 15:24:00.070 UTC [49] LOG:  database system was shut down at 2021-12-28 15:23:59 UTC
2021-12-28 15:24:00.076 UTC [48] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

2021-12-28 15:24:00.318 UTC [48] LOG:  received fast shutdown request
waiting for server to shut down....2021-12-28 15:24:00.320 UTC [48] LOG:  aborting any active transactions
2021-12-28 15:24:00.323 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
2021-12-28 15:24:00.323 UTC [50] LOG:  shutting down
2021-12-28 15:24:00.340 UTC [48] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2021-12-28 15:24:00.450 UTC [1] LOG:  starting PostgreSQL 13.5 (Debian 13.5-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
2021-12-28 15:24:00.450 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2021-12-28 15:24:00.450 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2021-12-28 15:24:00.454 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2021-12-28 15:24:00.460 UTC [62] LOG:  database system was shut down at 2021-12-28 15:24:00 UTC
2021-12-28 15:24:00.467 UTC [1] LOG:  database system is ready to accept connections
2022-01-01 19:09:47.623 UTC [1] LOG:  received fast shutdown request
2022-01-01 19:09:47.636 UTC [1] LOG:  aborting any active transactions
2022-01-01 19:09:47.647 UTC [1] LOG:  background worker "logical replication launcher" (PID 68) exited with exit code 1
2022-01-01 19:09:47.659 UTC [63] LOG:  shutting down
2022-01-01 19:09:47.748 UTC [1] LOG:  database system is shut down
time="2022-01-01T22:09:48+03:00" level=error msg="error waiting for container: unexpected EOF"
The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data ... ok
creating subdirectories ... ok
selecting dynamic shared memory implementation ... posix
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting default time zone ... Etc/UTC
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok


Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data -l logfile start

initdb: warning: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
waiting for server to start....2022-01-05 20:36:26.001 UTC [49] LOG:  starting PostgreSQL 13.5 (Debian 13.5-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
2022-01-05 20:36:26.008 UTC [49] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2022-01-05 20:36:26.028 UTC [50] LOG:  database system was shut down at 2022-01-05 20:36:25 UTC
2022-01-05 20:36:26.039 UTC [49] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

2022-01-05 20:36:26.338 UTC [49] LOG:  received fast shutdown request
waiting for server to shut down....2022-01-05 20:36:26.344 UTC [49] LOG:  aborting any active transactions
2022-01-05 20:36:26.346 UTC [49] LOG:  background worker "logical replication launcher" (PID 56) exited with exit code 1
2022-01-05 20:36:26.347 UTC [51] LOG:  shutting down
2022-01-05 20:36:26.384 UTC [49] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2022-01-05 20:36:26.477 UTC [1] LOG:  starting PostgreSQL 13.5 (Debian 13.5-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
2022-01-05 20:36:26.477 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2022-01-05 20:36:26.477 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2022-01-05 20:36:26.489 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2022-01-05 20:36:26.504 UTC [63] LOG:  database system was shut down at 2022-01-05 20:36:26 UTC
2022-01-05 20:36:26.515 UTC [1] LOG:  database system is ready to accept connections
2022-01-05 21:59:12.211 UTC [304] ERROR:  syntax error at or near "dbServiceFirst" at character 1
2022-01-05 21:59:12.211 UTC [304] STATEMENT:  dbServiceFirst
2022-01-10 06:53:23.511 UTC [1] LOG:  received fast shutdown request
2022-01-10 06:53:23.550 UTC [1] LOG:  aborting any active transactions
2022-01-10 06:53:23.553 UTC [304] FATAL:  terminating connection due to administrator command
2022-01-10 06:53:23.559 UTC [1] LOG:  background worker "logical replication launcher" (PID 69) exited with exit code 1
2022-01-10 06:53:23.560 UTC [64] LOG:  shutting down
2022-01-10 06:53:23.720 UTC [1] LOG:  database system is shut down
time="2022-01-10T09:53:24+03:00" level=error msg="error waiting for container: unexpected EOF"
The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data ... ok
creating subdirectories ... ok
selecting dynamic shared memory implementation ... posix
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting default time zone ... Etc/UTC
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok


Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data -l logfile start

initdb: warning: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
waiting for server to start....2022-01-11 09:19:41.093 UTC [49] LOG:  starting PostgreSQL 13.5 (Debian 13.5-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
2022-01-11 09:19:41.099 UTC [49] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2022-01-11 09:19:41.119 UTC [50] LOG:  database system was shut down at 2022-01-11 09:19:40 UTC
2022-01-11 09:19:41.129 UTC [49] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

2022-01-11 09:19:41.416 UTC [49] LOG:  received fast shutdown request
waiting for server to shut down....2022-01-11 09:19:41.421 UTC [49] LOG:  aborting any active transactions
2022-01-11 09:19:41.423 UTC [49] LOG:  background worker "logical replication launcher" (PID 56) exited with exit code 1
2022-01-11 09:19:41.424 UTC [51] LOG:  shutting down
2022-01-11 09:19:41.461 UTC [49] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2022-01-11 09:19:41.551 UTC [1] LOG:  starting PostgreSQL 13.5 (Debian 13.5-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
2022-01-11 09:19:41.551 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2022-01-11 09:19:41.551 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2022-01-11 09:19:41.562 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2022-01-11 09:19:41.576 UTC [63] LOG:  database system was shut down at 2022-01-11 09:19:41 UTC
2022-01-11 09:19:41.586 UTC [1] LOG:  database system is ready to accept connections
2022-01-11 13:00:40.569 UTC [708] LOG:  unexpected EOF on client connection with an open transaction
2022-01-11 13:28:53.937 UTC [791] LOG:  unexpected EOF on client connection with an open transaction
2022-01-11 13:41:03.169 UTC [808] LOG:  unexpected EOF on client connection with an open transaction
2022-01-11 13:58:59.057 UTC [894] LOG:  unexpected EOF on client connection with an open transaction
2022-01-11 14:25:59.689 UTC [960] LOG:  unexpected EOF on client connection with an open transaction
2022-01-11 14:25:59.739 UTC [971] LOG:  unexpected EOF on client connection with an open transaction
2022-01-11 14:51:41.378 UTC [979] LOG:  unexpected EOF on client connection with an open transaction
2022-01-11 15:00:05.701 UTC [1041] LOG:  unexpected EOF on client connection with an open transaction
2022-01-11 15:01:25.685 UTC [1064] LOG:  unexpected EOF on client connection with an open transaction
2022-01-11 15:23:28.281 UTC [1120] LOG:  unexpected EOF on client connection with an open transaction
2022-01-11 15:24:45.963 UTC [1134] LOG:  unexpected EOF on client connection with an open transaction
2022-01-11 15:29:01.383 UTC [1142] LOG:  unexpected EOF on client connection with an open transaction
2022-01-11 15:29:42.677 UTC [1155] LOG:  unexpected EOF on client connection with an open transaction
2022-01-11 15:33:29.692 UTC [1164] LOG:  unexpected EOF on client connection with an open transaction
2022-01-11 15:40:07.206 UTC [1185] LOG:  unexpected EOF on client connection with an open transaction
